### PR TITLE
Chore(image_push): Add script to push litmus image to the target image registry

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <udit@chaosnative.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Jonsy13 — DISMISSED** [review]

**Review by oumkale — APPROVED** [review]

File: custom/scripts/push_images/README.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
+## Table of content
+- [Supported Tunables](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#supported-tunables)
```
Comment (ispeakc0de) [inline custom/scripts/push_images/README.md:2]: Can we validate the links? 
push_images#supported-tunables or push_images/README.md#supported-tunables

File: custom/scripts/push_images/README.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+- [Check the available commands](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#check-the-available-commands)
+- [List Down all the Images](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#list-down-all-the-images)
+- [Pull the LitmusChaos Images into your machine](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#pull-the-litmuschaos-images-into-your-machine)
+- [Push the LitmusChaos Images to your image registry](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#push-the-litmuschaos-images-to-your-image-registry)
+
+## LitmusChaos Images
+
+- Pish docker images contains all the images that are used to execute a litmuschaos generic experiment using litmus portal. For more information please check [LitmusChaos Docs](https://litmusdocs-beta.netlify.app/docs/introduction).
```
Comment (ispeakc0de) [inline custom/scripts/push_images/README.md:10]:
```suggestion
- Push docker images contains all the images that are used to execute a litmuschaos generic experiment using litmus portal. For more information please check [LitmusChaos Docs](https://litmusdocs-beta.netlify.app/docs/introduction).
```

File: custom/scripts/push_images/README.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+- For pulling the litmus image and pushing into your registry please follow the given steps:
+
+```bash
+wget https://raw.githubusercontent.com/litmuschaos/test-tools/master/custom/scripts/push_images/litmus_image_push.sh
+chmod +x litmus_image_push.sh
+```
+
+#### Supported Tunables
+
+ <table>
```
Comment (ispeakc0de) [inline custom/scripts/push_images/README.md:24]: Can we create a table to list all the portal_images, backend_images, and workflow_images inside README?

File: custom/scripts/push_images/README.md (hunk 1 above)
Comment (uditgaurav) [reply custom/scripts/push_images/README.md]: Have updated. We can update with the actual links in https://github.com/litmuschaos/test-tools/pull/178 PR.

**Review by ispeakc0de — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

**Pull request merged**